### PR TITLE
Implement base 10 logarithm

### PR DESCRIPTION
**Pull request by sduquemesa**
**Context:**
There is no log10 function in tensorflow, only the loge exist ([ref](https://github.com/tensorflow/tensorflow/issues/1666))

**Description of the Change:**
Implements log base 10.

**Benefits:**
Well... a 10/10 implementation of log_10

**Possible Drawbacks:**
Could use `tf.function` to compile it but this function is not used in any performance-critical code so far.

**Related GitHub Issues:**
None

### CODE REVIEW

**Review by ziofil — APPROVED** [review]

**Comment by ziofil** [comment]
@sduquemesa this too, it was approved, see if we still need it

**Comment by sduquemesa** [comment]
This is useful but not a deal breaker if not added to the math backend. I'll leave it to you to decide whether to add this change or close this PR.

**Pull request closed without merging**
